### ISSUE TITLE
[NEWS] 30 września usuniemy POST /pricing/fee-preview / On September 30th we will remove POST /pricing/fee-preview

### ISSUE THREAD

**Issue by AureliuszBrussy** [allegro/allegro-api#4728]
**30 września 2021** usuniemy [POST /pricing/fee-preview](https://developer.allegro.pl/documentation/#operation/previewFeesPublicAPIUsingPOST). Zasób ten pozwala na podstawie określonych warunków oferty sprawdzić, jakie opłaty pobieramy w danej kategorii.

Skorzystaj z nowszego [POST /pricing/offer-fee-preview](https://developer.allegro.pl/documentation/#operation/calculateFeePreviewUsingPOST), dzięki któremu sprawdzisz, jakie opłaty naliczymy w ramach konkretnej oferty. Wystarczy, że w strukturze requesta przekażesz pełne dane oferty.

**Dlaczego usuwamy zasób?**

[POST /pricing/fee-preview](https://developer.allegro.pl/documentation/#operation/previewFeesPublicAPIUsingPOST) korzysta z pól, które nie mają aktualnie wpływu na naliczane opłaty. Jednocześnie **/pricing/offer-fee-preview** zastępuje zasób **/pricing/fee-preview** - na podstawie konkretnych danych oferty otrzymasz precyzyjną informację, jakie opłaty w ramach jej wystawienia naliczymy.

Więcej informacji o [POST /pricing/offer-fee-preview](https://developer.allegro.pl/documentation/#operation/calculateFeePreviewUsingPOST) znajdziesz w [naszym poradniku](https://developer.allegro.pl/charges/#kalkulator-op%C5%82at).

-------
On **September 30th, 2021** we will remove [POST /pricing/fee-preview](https://developer.allegro.pl/en/documentation/#operation/previewFeesPublicAPIUsingPOST). This resource allows you to check fees we charge in a given category based on the specified conditions of the offer.

Use newer [POST /pricing/offer-fee-preview](https://developer.allegro.pl/en/documentation/#operation/calculateFeePreviewUsingPOST), which allows you to see fees we will charge for a specific offer. All you need to do is provide the full body of the offer in the request structure.

**Why are we removing a resource?**

[POST /pricing/fee-preview](https://developer.allegro.pl/en/documentation/#operation/previewFeesPublicAPIUsingPOST) uses fields that currently do not affect the fees charged. At the same time, **/pricing/offer-fee-preview** replaces the **/pricing/fee-preview** resource - based on specific offer data, you will receive precise information about what listing fees we will charge.

For more information about [POST /pricing/offer-fee-preview](https://developer.allegro.pl/en/documentation/#operation/calculateFeePreviewUsingPOST) see our [guide](https://developer.allegro.pl/en/charges/#fee-calculator).

**Comment by jakubmaguza** [comment]
Ile danych z oferty muszę podać, żeby dostać informację o prowizji? Warunki zwrotów raczej nie mają na to wpływu.

**Comment by AureliuszBrussy** [comment]
Tak, masz rację - w [POST /pricing/offer-fee-preview](https://developer.allegro.pl/documentation/#operation/calculateFeePreviewUsingPOST) nie wszystkie dane oferty muszą zostać podane, aby prawidłowo wyliczyć opłaty. Niezbędne jest przede wszystkim wypełnienie pól lub struktur: **category.id**, **sellingMode**, **publication**, **promotion**. 
Zamysł tego zasobu jest jednak taki, aby móc kalkulować opłaty oferty w pełni gotowej do aktywowania.

**Comment by TimothyKoval** [comment]
Zamysł jest też widzę taki, że trzeba zrobić upload zdjęć żeby poznać koszty prowizji. To super, bo poprzedni endpoint był za szybki, zawsze mnie to denerwowało...

**Comment by AureliuszBrussy** [comment]
@TimothyKoval nie, nie musisz w w POST /pricing/offer-fee-preview przekazywać zdjęć aby poznać koszt prowizji. Możesz pominąć pole **images** lub przekazać pustą tablicę.
